massgov/mayflower-patternlab
# DP-8436 / DP-8439: If you add information details to a binder, ... and [MF] When you use TOC on the right side of info details

Any PRs being created needs a changelog.txt file before being merged into dev. See: [Change Log Instructions](https://github.com/massgov/mayflower/blob/dev/docs/change-log-instructions.md)


## Description

This moves the post content section of the information details template to match other templates.  This allows the post content section to be used for the binder navigation, making it full width instead of well width.

This also addresses the sticky TOC text that is displayed after a link is clicked in the sidebar TOC.

## Related Issue / Ticket

- [JIRA issue DP-8436](https://jira.state.ma.us/browse/DP-8436)
- [JIRA issue DP-8439](https://jira.state.ma.us/browse/DP-8439)

## Steps to Test


1. Information details pages still match previous versions.
    1. You should be able to confirm this in the visual regression image changes by following these steps:
        1. Go to "Files changed" tab above
        1. Find the updated images 
            1. regression_template_01-content-types-information-details_0_document_0_phone.png 
            1. regression_template_01-content-types-information-details_0_document_1_tablet.png
        1. On each file, click on the document icon that shows "Display the rich diff" when you hover over it.
    1. It would also be good to look at the Information details pages deployed to my Github Pages:
        1.  https://tnathanjames.github.io/mayflower/?p=pages-information-details
        1. https://tnathanjames.github.io/mayflower/?p=pages-information-details-preContent-media
        1. https://tnathanjames.github.io/mayflower/?p=pages-information-details-table-of-contents
1. When selecting an item from the sticky TOC, you should see the heading in the sticky header on Desktop.  Tablet and Mobile versions do not show this text.
    1. Go to the Information Details Table of Contents page: https://tnathanjames.github.io/mayflower/?p=pages-information-details-table-of-contents.
    1. Scroll past the Table of contents so that the sticky header appears.
    1. Click on the Table of contents button to the top right.
    1. Select any link from the Table of contents.
    1. Confirm that the Text you clicked appears in the header.

## Screenshots
Use something like [licecap](http://www.cockos.com/licecap/) to capture gifs to demonstrate behaviors.
<img width="549" alt="info_details_toc" src="https://user-images.githubusercontent.com/5727673/38263821-ba3e41c8-3736-11e8-97fc-6e7b7dc13a77.png">

<img width="881" alt="info_details_toc_destination_shows" src="https://user-images.githubusercontent.com/5727673/38263832-c4754614-3736-11e8-8cf3-0573ec2fd73d.png">


## Additional Notes:

Anything else to add?

#### Impacted Areas in Application


* None

#### @TODO


*

#### Today I learned...
